otter/convergence/service.py

```diff
@@ -415,9 +419,9 @@ def __init__(self, log, dispatcher, buckets, partitioner_factory,
             create an :obj:`Partitioner` to distribute the buckets.
         """
         MultiService.__init__(self)
+        self.log = log.bind(system='converger')
```

Comment:
Could you add `otter_service` back? It was an intentional decision to use `otter_service` instead of `system` because `system` gets overridden so much that a lot of context gets lost in the logs.